src/vivarium_public_health/disease/transition.py

```diff
@@ -28,6 +28,9 @@ def __new__(cls, value):
         obj.from_state, obj.to_state = value.split("_TO_")
         return obj
 
+    def __getnewargs__(self):
+        return (self.from_state + "_TO_" + self.to_state,)
```

Comment:
I wonder if this would have solved the issue I was running into where rq couldn't deserialize a TransitionString

Reply:
Does this solve this ticket? https://jira.ihme.washington.edu/browse/MIC-5149

Reply:
I didn't run into this issue while doing my own MNCH runs; other than that I'm not sure how to reproduce this.